src/js/components/RoutedButton/RouterContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const RouterContext = React.createContext();
```

Comment:
do we need both these files? can we just have one `RouterContext` under our context folder?

Reply:
Yeah I was considering having one RouterContext within the context folder, the reason I ended up creating two RouterContexts and putting them within the RoutedButton and RoutedAnchor was just to keep them a bit more contained and tied to those specific components. Since RoutedButton and RoutedAnchor are deprecated I wanted to keep the RouterContext relatively contained and nearby to those components so that it doesn't have as much visibility as something in the context folder might have. Just a soft opinion though, I'm good either way though if we want to move it to the context folder.

Reply:
gotcha no that makes sense that way if we get rid of those files I understand your thinking..